### PR TITLE
Add ES-notebook as script

### PR DESCRIPTION
**Pull request by thomasht86**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

- [x] Used `filter_path` to get only needed fields from response
- [x] Use `app_es/index-settings.json` for config. 
- [x] Made the notebook (script) self-contained (except from index settings)  
- [x] Use threading to parallellize uploading. 
- [x] Upload chunks directly from decompress stream to avoid unzipping large file. 
- [x] Added dynamic download from S3-url. (Not tested that all configurations are present there yet)

Ignore the 'run_benchmarks_vespa.py' for now, as it is WIP. 

- Removed the notebook now @geirst

### CODE REVIEW

**Review by geirst — COMMENTED** [review]
I have only reviewed `run_es_benchmarks.py`. Consider removing `run_vespa_benchmarks.py` from this PR until it is complete.

File: tests/performance/ecommerce_hybrid_search/common/run_es_benchmarks.py [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+            "-s", # Seconds to run the benchmark
+            "5",
+            "-n", # Number of clients in parallel
+            "1",
+            "-q", # Query file
+            config["query_file"],
+            "-P", # Post requests (GET is default)
+            "-o", # Output file
+            config["output_file"],
+            "-D", # Use TLS configuration from environment
```
[the remark this answers is not in the document]
Comment (thomasht86) [reply tests/performance/ecommerce_hybrid_search/common/run_es_benchmarks.py:332]: it worked in docker, but better to remove it then.

**Comment by thomasht86** [comment]
Added the script for running vespa benchmarks as script and removed notebook. 
Delaying refactoring between notebooks for now, as it might be a wasted effort, until we know what we are doing with this.

File: tests/performance/ecommerce_hybrid_search/common/run_vespa_benchmarks.py [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+if not os.path.exists(feed_file_vespa.replace(".zst", "")):
+    unzstd(feed_file_vespa)
+
+# %%
+import subprocess
+
+# Use vespa CLI (vespa feed --progress 10) to feed data
+set_target_command = "vespa config set target local"
+feed_command = f"vespa feed --progress 10 {feed_file_vespa.replace('.zst', '')}"
```
Comment (geirst) [inline tests/performance/ecommerce_hybrid_search/common/run_vespa_benchmarks.py:172]: Ideally we should use `vespa-feed-client` to be on par with the performance test, but we can keep this for now.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**